src/Core/Services/Logging/Logger.cs

```diff
@@ -29,6 +40,60 @@ protected Logger()
         {
         }
 
+
+        public string Description
+        {
+            get
+            {
+                return JsonConvert.SerializeObject(new
+                {
+                    AppId = _appId,
+                    UserId = _userId
+                }, Formatting.Indented);
+            }
+        }
+
+        public async Task InitAsync()
+        {
+            if (_isInitialised)
+            {
+                return;
+            }
+
+            var device = ServiceContainer.Resolve<IPlatformUtilsService>("platformUtilsService").GetDevice();
+            _userId = await ServiceContainer.Resolve<IStateService>("stateService").GetActiveUserIdAsync();
+            _appId = await ServiceContainer.Resolve<IAppIdService>("appIdService").GetAppIdAsync();
+
+            switch (device)
+            {
+                case Enums.DeviceType.Android:
+                    AppCenter.Start(DroidAppSecret, typeof(Crashes));
+                    break;
+                case Enums.DeviceType.iOS:
+                    AppCenter.Start(iOSAppSecret, typeof(Crashes));
+                    break;
+                default:
+                    return;
+
+            }
```

Comment:
Ye i think this is the case that needs an exception. If someone adds a new device type in the future, for any reason we cannot predict now, it would be secretly sending reports to the "else case" of the if and it would be pretty hard to notice it.